src/js/index.js

```diff
@@ -47,7 +47,10 @@ const hashTitles = {
     "#saved-sets": "Saved Sets",
     "#admin": "Admin Portal",
     "#search": "Browse Sets",
-    "#account": "My Account"
+    "#account": "My Account",
+    "#Discord":"Discord",
+    "#Donate": "Donate"
+    "#Youtube": "Youtube"
```

Comment:
Looks good, just make the hashes lowercase and make sure there are commas after every element that is not the last one
```suggestion
    "#discord":"Discord",
    "#donate": "Donate",
    "#youtube": "Youtube"
```

Reply:
But YouTube is not a page